src/main/java/seedu/duke/Duke.java

```diff
@@ -5,18 +5,34 @@
 public class Duke {
     public static void main(String[] args) {
 
-        System.out.println("Hello! I'm FridgeFriend!");
-        System.out.println("What can I do for you?\n");
+        welcomeMessage();
 
         String input;
         Scanner in = new Scanner(System.in);
         input = in.nextLine();
-        while (!input.equals("bye")){
-            System.out.println(input + "\n");
-            input = in.nextLine();
+
+        boolean isBye = false;
+        while (!isBye) {
+            switch (input) {
```

Comment:
Would it be better to consider the input command as a lower case to handle the upper case command of bye?

Reply:
resolved!